x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -3456,8 +3456,6 @@
     "xpack.apm.settings.agentConf.configTable.serviceNameColumnLabel": "サービス名",
     "xpack.apm.settings.agentConf.configurationsPanelTitle": "構成",
     "xpack.apm.settings.agentConf.createConfigButtonLabel": "構成の作成",
-    "xpack.apm.settings.agentConf.pageTitle": "設定",
-    "xpack.apm.settings.agentConf.returnToOverviewLinkLabel": "概要に戻る",
```

Comment:
random question: why is it that it's always only Japanese and Chinese translations that needs updating?